lib/core/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
```

Comment:
Are you sure about this? It's weird to see our browserlist supports IE6 but the TS is configured to ESNext